packages/amplify-graphql-relational-transformer/src/__tests__/__snapshots__/amplify-graphql-relations-with-custom-primary-key.test.ts.snap

```diff
@@ -730,20 +752,23 @@ exports[`Resolvers for custom primary key and relational directives should not g
 "#if( $ctx.stash.deniedField )
   #return($util.toJson(null))
 #end
-#if( $util.isNull($ctx.source.postId) )
+#set( $partitionKeyValue = $util.defaultIfNull($ctx.stash.connectionAttributes.get(\\"postId\\"), $ctx.source.postId) )
+#if( $util.isNull($partitionKeyValue) )
   #set( $result = {
   \\"items\\":   []
 } )
   #return($result)
 #else
   #set( $limit = $util.defaultIfNull($context.args.limit, 100) )
+  #set( $sortKeyValue0 = $util.defaultIfNull($ctx.stash.connectionAttibutes.get(\\"title\\"), $ctx.source.title) )
+  #set( $sortKeyValue1 = $util.defaultIfNull($ctx.stash.connectionAttibutes.get(\\"likes\\"), $ctx.source.likes) )
```

Comment:
double check on usage of these sort key values please.

Reply:
Same as above as CPK feature flag is false